packages/core/src/Stores/traders-hub-store.js

```diff
@@ -395,6 +397,12 @@ export default class TradersHubStore extends BaseStore {
         this.setCombinedCFDMT5Accounts();
     }
 
+    get restricted_countries_filter_content() {
```

Comment:
Can you change it to a more better name maybe like `financial_restricted_countries` ?

Reply:
Done 😃 👍🏼